composer.json

```diff
@@ -13,22 +13,17 @@
   ],
   "description": "Similar to js-husky client git hook implementation",
   "require": {
+    "php": ">=7.2",
     "ext-json": "*",
-    "symfony/console": "~3.2",
-    "symfony/filesystem": "~3.4",
-    "ccinn/php-cs-fixer-config": "^0.1.0"
+    "ccinn/php-cs-fixer-config": "^0.1.0",
+    "symfony/console": "^5.1",
```

Comment:
bc break without any reason
you should go for ^3.4|^4|^5

Reply:
Yeah, you're right! Thanks!